GUI/src/cz/fidentis/gui/NavigatorTopComponent.java

```diff
@@ -480,4 +492,30 @@ public void clearSelectionIfNeeded(ProjectTopComponent project) {
             }
         }
     }
+
+    /**
+     * Creates a copy of given list of FacialPoints reversely transformed by given
+     * transformations. This is needed when displaying original FacialPoints after
+     * models were already registered by the transformations.
+     * @param points facial points of a model
+     * @param trans transformations used to the model
+     * @return copy of facial points reversely transformed by transformations.
+     */
+    private List<FacialPoint> applyReverseICPTrans(List<FacialPoint> points, List<ICPTransformation> trans) {

```

Comment:
good point, akurat neviem kde to dat. nasiel som triedu `cz.fidentis.processing.registration.Registrator` kde by sa to asi celkom hodilo ale neni to nikde pouzite :D pridam to asi tam kde su tie ostatne metody co aplikuju transformacie, nech je to aspon pokope